app.test.js

```diff
@@ -48,4 +49,40 @@ describe("GET /api", () => {
             expect(body).toHaveProperty("response")
         })
     })
+})
+
+
+describe.only("GET /api/articles/:article_id", () => {
```

Comment:
Make sure you are removing .onlys before you push code up, I can't be sure all the other tests are passing if a .only has been put on a describe block.